components/connector/internal/gqlschema/schema.graphql

```diff
@@ -0,0 +1,50 @@
+# Token
+type Token {
+    token: String! # eg.: "1edfc34g"
+}
+
+# Certificates
+type Certificates {
```

Comment:
I would think about better name for the type (I don't like the plural form), but `CertificationDetails` is all I can made up for now 😄 and I don't think it's is better than yours, so...

Reply:
Renamed to `CertificationResult`